Pod install error on v10.0.0-beta.10: Cannot find module download-realm.js

<!---

Questions: If you have questions about HOW TO use Realm, please ask on
StackOverflow: http://stackoverflow.com/questions/ask?tags=realm
We monitor the `realm` tag.

Feature Request: Just fill in the first two sections below.

Bugs: To help you as fast as possible with an issue please describe your issue
and the steps you have taken to reproduce it in as much detail as possible.

-->

## Goals

<!--- What are you trying to achieve? -->

Install Realm v10.0.0-beta.10 in a React Native app.

## Expected Results

<!--- What did you expect to happen? -->

Installing the iOS pods should download Realm and install successfully.

## Actual Results

<!--- What happened instead? -->
<!--- e.g. the stack trace of a crash -->

```
Installing RealmJS (10.0.0-beta.10)
[!] /bin/bash -c 
set -e
./scripts/xcode-download-realm.sh ./scripts

internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module '/Users/ioveracker/Source/my-app/node_modules/realm/scripts/scripts/download-realm.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)

error Command failed with exit code 1.
```

## Steps to Reproduce

<!--- What are steps we can follow to reproduce this issue? -->

1. Use yarn or npm to install `realm@10.0.0-beta.10` in a React Native project.
2. In the `ios` directory, run `pod install` to install CocoaPods.
3. Observe that the above error is thrown.

## Code Sample

<!---
Please provide a code sample or test case that highlights the issue.
If relevant, include your model definitions.
For larger code samples, links to external gists/repositories are preferred.
Full projects that we can compile and run ourselves are ideal!
-->

Basic React Native project that I used to reproduce this is available at [ioveracker/realm-v10.0.0-beta.10-pods-error](https://github.com/ioveracker/realm-v10.0.0-beta.10-pods-error).

## Version of Realm and Tooling

- Realm JS SDK Version: 10.0.0-beta.10
- Node or React Native: React Native 0.63.2
- Client OS & Version: macOS 10.15.5
- Which debugger for React Native: N/A

--- a/scripts/xcode-download-realm.sh
+++ b/scripts/xcode-download-realm.sh
@@ -29,6 +29,6 @@ if ! which node > /dev/null; then
   export NODE_COMMAND="nvm run $NODE_CURRENT_VERSION"
 fi
 
-$NODE_COMMAND "${SCRIPTS_DIR}/scripts/download-realm.js" ios --sync
+$NODE_COMMAND "${SCRIPTS_DIR}/download-realm.js" ios --sync
 
 exit 0

